packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/samples-dev/dataProductsDeleteSample.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { NetworkAnalyticsClient } from "@azure/arm-networkanalytics";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to delete data product resource.
+ *
+ * @summary delete data product resource.
+ * x-ms-original-file: 2023-11-15/DataProducts_Delete_MaximumSet_Gen.json
+ */
+async function dataProductsDeleteMaximumSetGen() {
+  const credential = new DefaultAzureCredential();
+  const subscriptionId = "00000000-0000-0000-0000-00000000000";
+  const client = new NetworkAnalyticsClient(credential, subscriptionId);
+  const result = await client.dataProducts.delete(
```

Comment:
can we just await delete ? I am not sure every delete has a result to return?

Reply:
i think this makes sense, updated.